xla/service/hlo_runner_interface.h

```diff
@@ -214,11 +214,15 @@ class HloRunnerInterface {
   static absl::StatusOr<std::unique_ptr<HloModule>> CreateModuleFromString(
       absl::string_view hlo_string, const DebugOptions& debug_options);
 
+  // Creates an HloModule from the given proto.
+  static absl::StatusOr<std::unique_ptr<HloModule>> HloProtoToModule(
```

Comment:
For consistency with the function above, can you rename to `CreateModuleFromProto`?